src/__tests__/data/decoratedModels/users.ts

```diff
@@ -30,8 +30,8 @@ export class User implements TigrisCollectionType {
 	@PrimaryKey({order: 1})
 	id: number;
 
-	@Field(TigrisDataTypes.DATE_TIME)
-	created: string;
+	@Field()
+	created: Date;
```

Comment:
This will be automatically inferred as `DATE_TIME` in schema.

Reply:
Can we automatically detect UUID too?

Reply:
UUID support is available through numerous 3rd party libraries, nothing built-in. [@types/uuid](https://www.npmjs.com/package/@types/uuid) is most commonly used, I'll add auto-inference on what UUID version is commonly used. At the moment, following is still possible:

```
import uuid;

@Field(TigrisDataTypes.UUID)
id: uuid;
```